src/Compiler/Driver/CompilerConfig.fs

```diff
@@ -517,6 +517,7 @@ type TcConfigBuilder =
 
         /// show times between passes?
         mutable showTimes: bool
+        mutable reportTimeToFile: string option
```

Comment:
`Time` -> `Times`, or is there a reason behind the singular form?

Reply:
Not really.
Especially waiting on proposals for the cmd switch itself, since this is what is visible. Then I will align the names in code with it.